purchase_no_rfq/models/purchase_order.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2021-Today GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    state = fields.Selection(selection_add=[("draft", "Draft"), ("sent", "Sent")])
```

Comment:
I would add a comment explaining we are renaming user labels here (I thought we were adding `draft` and `sent` states which are already defined in `purchase` module).